SnookR/sublist/admin.py

```diff
@@ -1,3 +1,9 @@
 from django.contrib import admin
+from sublist.models import Sublist
 
-# Register your models here.
+
+class SublistAdmin(admin.ModelAdmin):
+    pass
+
```

Comment:
@bobbyesh Does this distinction allow us to have `sublist` app specific admins, that aren't necessarily admins for the rest of the project?